DigitalLearningSolutions.Data.Tests/Services/GroupServiceTests/GroupsServiceEnrolDelegateTests.cs

```diff
@@ -588,6 +602,7 @@ public void EnrolDelegateOnGroupCourses_sends_correct_email_with_additional_Comp
             groupsService.EnrolDelegateOnGroupCourses(
                 oldDelegateDetails,
                 newAccountDetails,
+                null,

```

Comment:
Should we have an equivalent for this, but with a non-null centre email, to make sure it gets used here?